lib/database/set-claim.php

```diff
@@ -0,0 +1,500 @@
+<?php
+
+use RA\Permissions;
+
+/**
+ * Checks if the user is able to make a claim and inserts the claim into the database. If the claim
+ * is a collaboration claim then the claim will have the same Finished time as the primary claim for the game.
+ */
+function insertClaim(string $user, int $gameID, int $claimType, int $setType, int $special, int $permissions): bool
+{
+    sanitize_sql_inputs($user, $gameID, $claimType, $setType, $special);
+
+    if ($claimType == 0) { // Primary Claim
```

Comment:
Please create enums for magic constants.

Reply:
Updated to use enums.